.github/workflows/elixir.yml

```diff
@@ -0,0 +1,37 @@
+name: Elixir CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    name: Build and test
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Elixir
+      uses: erlef/setup-beam@988e02bfe678367a02564f65ca2e37726dc0268f
+      with:
+        elixir-version: '1.12.3'
```

Comment:
In our `mix.exs` we're using elixir 1.9, and the latest is 1.13.3. What do you think of upgrading this?

Reply:
Yeah, sounds perfect. But do you mean on both the `mix.exs` and yaml file? or just yaml file?

Reply:
Let's do the yaml file here, then we can have a different PR for the `mix.exs`